packages/blip/app/pages/app/app.js

```diff
@@ -183,23 +185,36 @@ export class AppComponent extends React.Component {
       }
     }
     if (config.HELP_LINK !== null && typeof window.zE === 'function') {
-      if (this.props.authenticated) {
-        let name = this.props.user.profile.fullName;
-        let email = this.props.user.emails[0];
+      this.renderZenDeskWidget();
+    }
+  }
 
+  renderZenDeskWidget() {
```

Comment:
it misses unit tests on this function (you could see the coverage report for more details)